docs/source/upgrading_your_components.md

```diff
@@ -29,8 +29,13 @@ This tutorial assumes a Docker deployment where the YAML files will be baked int
 
 When you deploy a peer or an ordering node, you had to set a number of environment variables relevant to its configuration. A best practice is to create a file for these environment variables, give it a name relevant to the node being deployed, and save it somewhere on your local file system. That way you can be sure that when upgrading the peer or ordering node you are using the same variables you set when creating it by issuing:
 
+For peer:
 ```
-docker run --env-file ./env<name of node>.list ubuntu bash
+docker run -d -v /opt/backup/$PEER_CONTAINER/:/var/hyperledger/production/ \
+            -v /opt/msp/:/etc/hyperledger/fabric/msp/ \
+            --env-file ./env<name of node>.list \
+            --name $PEER_CONTAINER \
+            hyperledger/fabric-peer peer node start
```

Comment:
Talked to Joe about the dismissed review comment. He was asking about the image versions in the command. It appears we use it in other places but not here.

Should we have a version?

Reply:
Talked to Surya. I'm fine with getting these merged for now, then I'll go back and add a bit more text and environment variables for these sections where needed in a subsequent PR.